cmd/shortener/main.go

```diff
@@ -1,3 +1,33 @@
 package main
 
-func main() {}
+import (
+	"net/http"
+
+	"mishin-shortener/internal/app/config"
+	"mishin-shortener/internal/app/handlers"
+	"mishin-shortener/internal/app/storage"
+
+	"fmt"
+	"github.com/go-chi/chi/v5"
+)
+
+func main() {
+
+	db := storage.Database{}
+	r := chi.NewRouter()
+
+	config.Parse()
+
+	r.Post("/", func(w http.ResponseWriter, r *http.Request) {
+		handlers.PostHandler(w, r, &db, config.Config.BaseRedirectURL)
+	})
+	r.Get("/{shortened}", func(w http.ResponseWriter, r *http.Request) {
+		handlers.GetHandler(w, r, &db)
+	})
+
+	fmt.Println("Server started on", config.Config.BaseServerURL)
```

Comment:
Рекомендую использовать log для логирования вместо `fmt.Println(...)`.

```suggestion
	log.Printf("Server started on %s", config.Config.BaseServerURL)
```

Reply:
Согласен, log гораздо информативнее выглядит. Исправлено